packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -664,10 +672,11 @@ class EnrollVerbHandler extends AbstractVerbHandler {
         break;
       case 'revoke':
       case 'deny':
+      case 'delete':
       case 'unrevoke':
         if (enrollParams!.enrollmentId.isNullOrEmpty) {
           throw AtEnrollmentException(
-              'enrollmentId is mandatory for enroll:revoke/enroll:deny');
+              'enrollmentId is mandatory for enroll:revoke/enroll:deny/enroll:delete');
```

Comment:
Maybe change to `for enroll:$operation`

Reply:
Changed made in [3ca6f91](https://github.com/atsign-foundation/at_server/pull/2095/commits/3ca6f9125f3bb4abbf71a69232624c730f6460b6)